docs/notes/ctl.rst

```diff
@@ -208,3 +209,18 @@ Example:
 .. code:: shell
 
     vineyard-ctl migrate --ipc_socket_value /tmp/vineyard.sock --object_id 00002ec13bc81226 --remote
+
+debug
+------
+
+Issue a debug request.
+
+Options:
+
++ :code:`payload`: The payload that will be sent to the debug handler.
+
+Example:
+
+.. code:: shell
+
+    vineyard-ctl debug --payload {"instance_status": [], "memory_size": []}
```

Comment:
This line will probably break, the argument can't have spaces, I suggest you can try yourself to ensure you can get the payload as you expected.

Reply:
Sure, I'll check this. :)

Reply:
Hi @siyuan0322, I have updated the example and checked to ensure everything works as expected.